phpstan.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
it looks like the content of the file is deleted
but file itself needs to be deleted instead

Reply:
I don't know why it looks like that, but file was deleted as expected. 🤔